src/swagger-specs/destinations.yaml

```diff
@@ -130,88 +368,624 @@ paths:
-      operationId: getFlows
+        You can delete the specified base connection by making a DELETE request to the `/connections` endpoint and providing the ID of the connection that you wish to delete in the request path.
+      operationId: deleteConnectionById
       parameters:
-        - in: header
-          name: Authorization
-          description: "The access token which can be copied from your Experience Platform integration, prefixed with 'Bearer '. For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en)."
-          schema:
-            type: string
-          required: true
-        - in: header
-          name: x-gw-ims-org-id
-          description: "The organization ID which can be copied from your Experience Platform integration. For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en)."
-          schema:
-            type: string
-          required: true
-        - in: header
-          name: x-sandbox-name
-          description: "All resources in Experience Platform are isolated to specific virtual sandboxes. Requests to Platform APIs require a header that specifies the name of the sandbox in which the operation will take place."
-          schema:
-            type: string
-          required: true
-        - in: header
-          name: x-api-key
-          description: "The API key which can be copied from your Experience Platform integration. For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en)."
-          schema:
-            type: string
+        - name: id
+          in: path
+          description: "The ID of the connection you are looking to delete." 
           required: true
-        - in: header
-          name: Accept
-          description: "An optional content negotiation header which allows you to use a specific version of the API. If you don't use it, the HTTP responses fall back to the latest version of the API."
           schema:
             type: string
-            default: "application/json; version=1"
-          required: false
+        - $ref: '#/components/parameters/Authorization'
+        - $ref: '#/components/parameters/XApiKey'
+        - $ref: '#/components/parameters/XImsOrgId'
+        - $ref: '#/components/parameters/XSandboxId'
+        - $ref: '#/components/parameters/Accept'
+      responses:
+        '202':
+          description: A successful response returns HTTP status 202 (No Content) and a blank body. You can confirm the deletion by attempting a lookup (GET) request to the connection. The API will return an HTTP 404 (Not Found) error, indicating that the connection has been deleted.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
```

Comment:
Looks like the `ErrorResponse` schema is a placeholder that needs to be filled out. When it comes to 202 responses though, there shouldn't be a `content` property since the status is literally "No Content" 😂 This should be fine with just a `description`